python/pip_install/extract_wheels/lib/requirements.py

```diff
@@ -1,6 +1,8 @@
 import re
 from typing import Dict, Optional, Set, Tuple
 
+import pkg_resources
```

Comment:
My comment doesn't intend to block this PR (I think we need this fix). 

Mostly an fyi for future travellers: `pkg_resources` (we use it elsewhere, so shouldn't block this PR anyway) is on the slow path towards deprecation: 
* https://github.com/pypa/setuptools/milestone/4
* https://setuptools.pypa.io/en/latest/pkg_resources.html

There are probably parts of the stdlib we could use in future: https://github.com/python/cpython/blob/9cc0afcb8782c22575fa91756c54029c7e774dfb/Lib/importlib/metadata/__init__.py#L867

Or just create our own function if that's easier.